### PR TITLE
Need better description for mach subcommands #12402

### LINKED ISSUE #12402: Need better description for mach subcommands
Currently the mach subcommand descriptions are not very clear, especially the following ones:

```
test-css              Run the web platform tests
test-wpt              Run the web platform tests
test-wpt-failure      Run the web platform tests
update-css            Update the web platform tests
update-wpt            Update the web platform tests
```

Perhaps someone who knows the difference can update them?

### PR DESCRIPTION
**Pull request by yoyo930021**
---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes fix #12402  (github issue number if applicable).

- [ ] There are tests for these changes OR
- [x] These changes do not require tests because is description

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/13087)

### CODE REVIEW

File: python/servo/testing_commands.py [hunk 1]
```diff
@@ -351,7 +351,7 @@ def test_wpt_failure(self):
         ], env=self.build_env())
 
     @Command('test-wpt',
-             description='Run the web platform tests',
+             description='Run the web platform tests that the regular test suite',
```
Comment (izgzhen) [inline python/servo/testing_commands.py]: Personally speaking, It feel a bit redundant for me, I would write something like "Run the regular web platform test suite", or just leave it as it is originally.

Comment (yoyo930021) [inline python/servo/testing_commands.py]: @izgzhen 
Thank!
My english is not well. ww
I modify.

File: python/servo/testing_commands.py [hunk 2]
```diff
@@ -338,7 +338,7 @@ def test_webidl(self, quiet, tests):
         return run_globals["run_tests"](tests, verbose)
 
     @Command('test-wpt-failure',
-             description='Run the web platform tests',
+             description='Run the small test harness that verifies that failures in tests are reported correctly',
```
Comment (Wafflespeanut) [inline python/servo/testing_commands.py]: Let's go for something like, `"Run the test harness that verifies that the test failures are reported correctly"`?

File: python/servo/testing_commands.py [hunk 3]
```diff
@@ -463,7 +463,7 @@ def update_jquery(self, release, dev):
         return self.jquery_test_runner("update", release, dev)
 
     @Command('test-css',
-             description='Run the web platform tests',
+             description='Run the css tests',
```
Comment (Wafflespeanut) [inline python/servo/testing_commands.py]: Nit: `"Run the web platform CSS tests"`

File: python/servo/testing_commands.py [hunk 4]
```diff
@@ -475,7 +475,7 @@ def _test_css(self, **kwargs):
         return self.wptrunner(run_file, **kwargs)
 
     @Command('update-css',
-             description='Update the web platform tests',
+             description='Update the css tests',
```
Comment (Wafflespeanut) [inline python/servo/testing_commands.py]: Nit: `"Update the web platform CSS tests"`

File: python/servo/testing_commands.py (hunk 2 above)
Comment (yoyo930021) [inline python/servo/testing_commands.py]: I try to modify.

File: python/servo/testing_commands.py (hunk 3 above)
Comment (yoyo930021) [inline python/servo/testing_commands.py]: I finished.
Thank.

File: python/servo/testing_commands.py (hunk 4 above)
Comment (yoyo930021) [inline python/servo/testing_commands.py]: I finished.
Thank.
@Wafflespeanut

File: python/servo/testing_commands.py [hunk 5]
```diff
@@ -338,7 +338,7 @@ def test_webidl(self, quiet, tests):
         return run_globals["run_tests"](tests, verbose)
 
     @Command('test-wpt-failure',
-             description='Run the web platform tests',
+             description='Run the small web platform failure test',
```
Comment (Wafflespeanut) [inline python/servo/testing_commands.py]: um, this wasn't my suggestion?

File: python/servo/testing_commands.py [hunk 6]
```diff
@@ -475,7 +475,7 @@ def _test_css(self, **kwargs):
         return self.wptrunner(run_file, **kwargs)
 
     @Command('update-css',
-             description='Update the web platform tests',
+             description='Update the web platform CSS test',
```
Comment (Wafflespeanut) [inline python/servo/testing_commands.py]: Nit: should be `"... tests"` (missing 's')

File: python/servo/testing_commands.py (hunk 5 above)
Comment (yoyo930021) [inline python/servo/testing_commands.py]: I finished.

File: python/servo/testing_commands.py (hunk 6 above)
Comment (yoyo930021) [inline python/servo/testing_commands.py]: I finished.
Thank!

File: python/servo/testing_commands.py [hunk 7]
```diff
@@ -338,7 +338,7 @@ def test_webidl(self, quiet, tests):
         return run_globals["run_tests"](tests, verbose)
 
     @Command('test-wpt-failure',
-             description='Run the web platform tests',
+             description='"Run the tests harness that verifies that the test failures are reported correctly',
```
Comment (Wafflespeanut) [inline python/servo/testing_commands.py]: Nit: There's a double quote at the start of the text.

**Comment by Wafflespeanut** [comment]
Also, while you're at this, can you ensure that all commands begin with title cases?

File: python/servo/testing_commands.py (hunk 7 above)
Comment (yoyo930021) [inline python/servo/testing_commands.py]: Oops! 
I finished.

**Comment by yoyo930021** [comment]
@Wafflespeanut 
I finished.

**Comment by Wafflespeanut** [comment]
Great work! Thanks :)

@bors-servo r+

**Comment by Wafflespeanut** [comment]
Actually, I'd said all "commands" instead of all command "descriptions" - good thing you caught it correctly :smile:

**Pull request merged**